src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -36,14 +35,18 @@ internal sealed class LocalFunctionSymbol : MethodSymbol
             Binder binder,
             Symbol containingSymbol,
             LocalFunctionStatementSyntax syntax)
+            : base(containingSymbol, syntax.GetReference(), syntax.ExpressionBody?.GetReference() ?? syntax.Body?.GetReference(), syntax.Location)
         {
             _syntax = syntax;
             _containingSymbol = containingSymbol;
 
-            _declarationModifiers =
-                DeclarationModifiers.Private |
-                DeclarationModifiers.Static |
-                syntax.Modifiers.ToDeclarationModifiers();
+            MakeFlags(
+                methodKind: MethodKind.LocalFunction,
+                declarationModifiers: DeclarationModifiers.Private |
+                                      DeclarationModifiers.Static |
+                                      syntax.Modifiers.ToDeclarationModifiers(),
+                returnsVoid: false, /* arbitrary, flags.ReturnsVoid is unused in local functions */
```

Comment:
> returnsVoid: false, /* arbitrary, flags.ReturnsVoid is unused in local functions */ [](start = 15, length = 84)

This doesn't feel right. This approach is fragile in the long run. Also, I believe ReturnsVoid is a public API, therefore, it is used, and it should return correct value.